packages/create-react-admin/src/app.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useReducer } from 'react';
+import { Box, Text } from 'ink';
+import {
+    InitialProjectConfiguration,
+    ProjectConfiguration,
+} from './ProjectState.js';
+import { generateProject } from './generateProject.js';
+import { StepDataProvider } from './StepDataProvider.js';
+import { StepAuthProvider } from './StepAuthProvider.js';
+import { StepResources } from './StepResources.js';
+
+type Props = {
+    name: string | undefined;
+};
+
+const stepReducer = (
+    state: ProjectConfiguration,
+    action
+): ProjectConfiguration => {
+    switch (state.step) {
+        case 'data-provider':
+            return {
+                ...state,
+                step: 'auth-provider',
+                dataProvider: action.value,
+                resources:
+                    action.value === 'ra-data-fakerest'
+                        ? ['posts', 'comments']
+                        : state.resources,
+            };
+        case 'auth-provider':
+            return {
+                ...state,
+                step:
+                    state.dataProvider === 'ra-data-fakerest'
+                        ? 'generate'
+                        : 'resources',
+                authProvider: action.value,
+            };
+        case 'resources':
+            return {
+                ...state,
+                step: 'generate',
+                resources: action.value,
+            };
+        case 'generate':
+            return {
+                ...state,
+                step: 'finish',
```

Comment:
We miss a final step where we ask the user if we should install the dependencies (choices: yarn / npm / ignore), and it should end with instructions to start the app (cd into the directory and run yarn start or npm start).

Reply:
Added